File: zarchiwizowane-z-1-klasy/1Egr2Tech19CS/KartyPracy/KartaPracy2a/KartaPracy2a.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace KartaPracy2a
{
    internal class Program
    {
    static void Main(string[] args)
        {
            Zadanie1(); Zadanie2(); Zadanie3(); Zadanie4(); Zadanie5(); Zadanie6();
        }
    static void Zadanie1()
        {
            System.Console.WriteLine("Zadanie 1: (easy) Sprawdź czy suma dwóch wpisanych przez użytkownika jest liczbą parzystą");
            System.Console.Write("Wpisz liczbę a: ");
            int a = int.Parse(Console.ReadLine());
            System.Console.Write("Wpisz liczbę b: ");
            int b = int.Parse(Console.ReadLine());
            
            
            if ((a+b)%2==0)
            {
                System.Console.WriteLine($"Suma liczb {a} i {b} jest parzysta");
            }
            else
            {
                System.Console.WriteLine($"Suma liczb {a} i {b} nie jest parzysta");
            }
            System.Console.Write("Naciśnij dowolny klawisz aby rozpocząć kolejne zadanie");
            Console.ReadKey();
            Console.Clear();
        }
    static void Zadanie2()
        {
            System.Console.WriteLine("Zadanie 2: (easy) Sprawdź, czy średnia arytmetyczna dwóch wpisanych liczb jest wieksza od jej średniej geometrycznej.");
            System.Console.Write("Wpisz liczbe a: ");
            int a = int.Parse(Console.ReadLine());
            System.Console.Write("Wpisz liczbe b: ");
            int b = int.Parse(Console.ReadLine());
            double ary = (a+b)/2;
            double geo = Math.Sqrt(a*b);

            if (ary>geo)
            {
                System.Console.WriteLine("Średnia arytmetyczna jest większa od średniej geometrycznej");
            }
            else if (ary<geo)
            {
                System.Console.WriteLine("Średnia arytmetyczna jest mniejsza od średniej geometrycznej");
            }
            else 
            {
                System.Console.WriteLine("Średnia arytmetyczna jest równa średniej geometrycznej");
            }
            System.Console.Write("Naciśnij dowolny klawisz aby rozpocząć kolejne zadanie");
            Console.ReadKey();
            Console.Clear();
        }
    static void Zadanie3()
        {
            System.Console.WriteLine("Zadanie 3: (easy) Użytkownik podaje 3 liczby całkowite. Sprawdź czy dokładnie dwie z nich są sobie równe");
            System.Console.Write("Wpisz liczbę a: ");
            int a = int.Parse(Console.ReadLine());
            System.Console.Write("Wpisz liczbę b: ");
            int b = int.Parse(Console.ReadLine());
            System.Console.Write("Wpisz liczbę c: ");
            int c = int.Parse(Console.ReadLine());
            
            if (a==b && !(a==c) && !(b==c))
            {
                System.Console.WriteLine("Tylko liczby a i b są siebie równe");
            }
            else if (a==c && !(a==b) && !(b==c))
            {
                System.Console.WriteLine("Tylko liczby a i c są siebie równe");
            }
             else if (b==c && !(a==c) && !(a==b))
            {
                System.Console.WriteLine("Tylko liczby c i b są siebie równe");
            }
            else
            {
                System.Console.WriteLine("W tym zbiorze żadna z liczb nie spełnia warunku");
            }

            System.Console.Write("Naciśnij dowolny klawisz aby rozpocząć kolejne zadanie");
            Console.ReadKey();
            Console.Clear();
        }
    static void Zadanie4()
        {
            System.Console.WriteLine("Zadanie 4: (medium) Niech użytkownik wprowadzi 4 różne liczby całkowite. Napisz program, który sprawdzi, która z tych liczb jest najmniejsza");
            System.Console.Write("Wpisz liczbę a: ");
            int a = int.Parse(Console.ReadLine());
            System.Console.Write("Wpisz liczbę b: ");
            int b = int.Parse(Console.ReadLine());
            System.Console.Write("Wpisz liczbę c: ");
            int c = int.Parse(Console.ReadLine());
            System.Console.Write("Wpisz liczbę d: ");
            int d = int.Parse(Console.ReadLine());
            
            if (a<b && a<c && a<d)
            {
                System.Console.WriteLine("Liczba A jest najmniejsza");
            }
            else if (b<a && b<c && b<d)
            {
                System.Console.WriteLine("Liczba B jest najmniejsza");
            }
             else if (c<a && c<b && c<d)
            {
                System.Console.WriteLine("Liczba C jest najmniejsza");
            }
              else if (d<a && d<b && d<c)
            {
                System.Console.WriteLine("Liczba D jest najmniejsza");
            }
            else 
            {
                System.Console.WriteLine("Użytkownik nie wprowadził różnych cyfr");
            }
            
            System.Console.Write("Naciśnij dowolny klawisz aby rozpocząć kolejne zadanie");
            Console.ReadKey();
            Console.Clear();
        }
    static void Zadanie5()
        {
            System.Console.WriteLine("Zadanie 5: Sprawdź czy trzy wprowadzone przez użytkownika liczby spełniają nierówność trójkąta");
            System.Console.Write("Wpisz liczbę a: ");
            int a = int.Parse(Console.ReadLine());
            System.Console.Write("Wpisz liczbę b: ");
            int b = int.Parse(Console.ReadLine());
            System.Console.Write("Wpisz liczbę c: ");
            int c = int.Parse(Console.ReadLine());

            if (a+b>=c || c+b>=a || a+c>=b)
            {
                System.Console.WriteLine("Liczby spełniają nierówność trójkąta");
            }            
            else
            {
                System.Console.WriteLine("Liczby nie spełniają nierówność trójkąta");
            }
            
            System.Console.Write("Naciśnij dowolny klawisz aby rozpocząć kolejne zadanie");
            Console.ReadKey();
            Console.Clear();
        }
    static void Zadanie6()
        {
            System.Console.WriteLine("Zadanie 6 : (medium) Niech z trzech wprowadzonych przez użytkwnika liczbach da się zbudować trójąt. Sprawdź czy ten trójką jest ostrokątny, prostokątny czy rozwartokątny");
            System.Console.Write("Podaj kąt przy wierzchołku A: ");
            int a = int.Parse(Console.ReadLine());
            System.Console.Write("Podaj kąt przy wierzchołku B: ");
            int b = int.Parse(Console.ReadLine());
            System.Console.Write("Podaj kąt przy wierzchołku C: ");
            int c = int.Parse(Console.ReadLine());
           
          if ((a < 90 && b < 90 && c < 90) && a+b+c == 180)
          {
            System.Console.WriteLine("Z danych kątów da się zrobić trójkąt ostrokątny");
          }
          else if (((a == 90  && ( b < 90 && c < 90)) || (b == 90  && ( a < 90 && c < 90)) || (c == 90  && ( b < 90 && a < 90))) && a+b+c == 180)
          {
            System.Console.WriteLine("Z danych kątów da się zrobić trójkąt prostokątny");
          }
          else if (((a > 90  && ( b < 90 && c < 90)) || (b > 90  && ( a < 90 && c < 90)) || (c > 90  && ( b < 90 && a < 90))) && a+b+c == 180)
          {
            System.Console.WriteLine("Z danych kątów da się zrobić trójkąt rozwartokątny");
          }
          else 
          {
            System.Console.WriteLine("Z danych kątów nie da się zrobić trójkąta");
          }
        }
    }   
}

File: zarchiwizowane-z-1-klasy/1Eg2TechPython/KartyPracy/KartaPracy2a/KartaPracy2a.py

# Zadanie 1
print("Zadanie 1: (easy) Sprawdź czy suma dwóch wpisanych przez użytkownika jest liczbą parzystą")

print("Wpisz liczbe a:")
a = int(input())
print("Wpisz liczbe b:")
b = int(input())
if (a+b)%2==0:
    print("Suma a i b jest liczbą parzystą")
else:
    print("Suma a i b nie jest liczbą parzystą")
# Zadanie 2
print("Zadanie 2: (easy) Sprawdź, czy średnia arytmetyczna dwóch wpisanych liczb jest wieksza od jej średniej geometrycznej")
print("Wpisz liczbe a:")
a = int(input())
print("Wpisz liczbe b:")
b = int(input())
art = (a+b)/2
from math import sqrt
geo = (sqrt(a*b))
if art > geo :
    print("Średnia arytmetyczna jest większa od średniej geometrycznej")
else: 
 if geo < art:
    print("Średnia arytmetyczna jest mniejsza od średniej geometrycznej") 
 else:
  if geo == art:
    print("Średnia arytmetyczna jest równa średniej geometrycznej")
# Zadanie 3
print("Zadanie 3: (easy) Użytkownik podaje 3 liczby całkowite. Sprawdź czy dokładnie dwie z nich są sobie równe")
print("Wpisz liczbe a:")
a = int(input())
print("Wpisz liczbe b:")
b = int(input())
print("Wpisz liczbe c:")
c = int(input())
if (a==b) and (a!=c) and (b!=c):
    print("Tylko liczby a i b są siebie równe")
else:
 if (a==c) and (a!=b) and (b!=c):
    print("Tylko liczby a i c są siebie równe")
 else:
  if (b==c) and (c!=a) and (b!=a):
    print("Tylko liczby c i b są siebie równe")
  else:
   print("W tym zbiorze żadna z liczb nie spełnia warunku")
# Zadanie 4
print("Zadanie 4: (medium) Niech użytkownik wprowadzi 4 różne liczby całkowite. Napisz program, który sprawdzi, która z tych liczb jest najmniejsza")
print("Wpisz liczbe a:")
a = int(input())
print("Wpisz liczbe b:")
b = int(input())
print("Wpisz liczbe c:")
c = int(input())
print("Wpisz liczbe d:")
d = int(input())
if a<b and a<c and a<d:
 print("Liczba A jest najmniejsza")
else:
 if b<a and b<c and b<d:
  print("Liczba B jest najmniejsza")
 else:
  if c<a and c<b and c<d:
   print("Liczba C jest najmniejsza")
  else:
   if d<a and d<b and d<c:
    print("Liczba D jest najmniejsza")
   else:
    print("Użytkownik nie wprowadził różnych cyfr")
# Zadanie 5
print("Zadanie 5: Sprawdź czy trzy wprowadzone przez użytkownika liczby spełniają nierówność trójkąta")
print("Wpisz liczbe a:")
a = int(input())
print("Wpisz liczbe b:")
b = int(input())
print("Wpisz liczbe c:")
c = int(input())
if (a+b>=c) or (c+b>=a) or (a+c>=b):
 print("Liczby spełniają nierówność trójkąta")
else:
 print("Liczby nie spełniają nierówność trójkąta")
# Zadanie 6
print("Zadanie 6 : (medium) Niech z trzech wprowadzonych przez użytkwnika liczbach da się zbudować trójąt. Sprawdź czy ten trójką jest ostrokątny, prostokątny czy rozwartokątny")
print("Podaj kąt przy wierzchołku A:")
a = int(input())
print("Podaj kąt przy wierzchołku B:")
b = int(input())
print("Podaj kąt przy wierzchołku C:")
c = int(input())
if (a < 90 and b < 90 and c < 90) and a+b+c == 180:
 print("Z danych kątów da się zrobić trójkąt ostrokątny")
else:
 if ((a == 90  and ( b < 90 and c < 90)) or (b == 90  and ( a < 90 and c < 90)) or (c == 90  and ( b < 90 and a < 90))) and a+b+c == 180:
  print("Z danych kątów da się zrobić trójkąt prostokątny")
 else:
  if ((a > 90  and ( b < 90 and c < 90)) or (b > 90  and ( a < 90 and c < 90)) or (c > 90  and ( b < 90 and a < 90))) and a+b+c == 180:
   print("Z danych kątów da się zrobić trójkąt rozwartokątny")
  else:
   print("Z danych kątów nie da się zrobić trójkąta")         